examples/hooks/App.tsx

```diff
@@ -117,8 +119,15 @@ export function App() {
             )}
           </QueryRuleCustomData>
 
-          <Hits hitComponent={Hit} />
-          <Pagination className="Pagination" />
+          <Tabs>
+            <Tab title="Show Hits">
+              <Hits hitComponent={Hit} />
+              <Pagination className="Pagination" />
+            </Tab>
+            <Tab title="Show InfiniteHits">
```

Comment:
That's sufficient imo.

```suggestion
            <Tab title="Hits">
              <Hits hitComponent={Hit} />
              <Pagination className="Pagination" />
            </Tab>
            <Tab title="InfiniteHits">
```